modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -313,12 +320,6 @@
     /** Index create/rebuild pool size. */
     private int buildIdxPoolSize = DFLT_BUILD_IDX_THREAD_POOL_SIZE;
```

Comment:
Should this be a part of SqlConfiguration as well?

Reply:
Not agreed. All we configure and start all executor services at the IgnitionEx.IgniteNamedInstance#start0. So it looks looks like a core feature.